packages/shared/src/modules3rdParty/cross-crypto/verify.ts

```
@@ -1,28 +1,35 @@
 import assert from 'assert';
-import * as crypto from 'crypto';
+import * as nodeCrypto from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)